packages/terra-functional-testing/src/commands/utils/setLocale.js

```diff
@@ -0,0 +1,24 @@
+const { Logger } = require('@cerner/terra-cli');
+const dispatchCustomEvent = require('./dispatchCustomEvent');
+
+const logger = new Logger({ prefix: '[terra-functional-testing:setLocale]' });
+
+/**
+ * Updates terra-application's locale via custom event
+ * @param {string} locale - name of locale to update to
+ */
+const setLocale = (locale) => {
+  if (typeof locale === 'undefined' || !locale) {
```

Comment:
The second check would probably cover both cases here, but nothing wrong with being extra precautions. 

![image](https://user-images.githubusercontent.com/6720991/104224179-39240480-540a-11eb-839e-38e4d1f80ebd.png)